bin/specextract

```diff
@@ -783,14 +812,14 @@ def pi_alternate_wmap(pi,wmap):
     dmcopy.punlearn()
     dmcopy.infile = tmp_wmap.name
     dmcopy.outfile = pi
-    dmcopy.clobber = "yes"
-    dmcopy.verbose = "0"
+    dmcopy.clobber = True
+    dmcopy.verbose = 0
     dmcopy()
     
     dmappend.punlearn()
```

Comment:
Has this been tested? It doesn't look dmappend has been imported - e.g. it's not in

```python
from ciao_contrib.runtool import dmextract, mkrmf, mkacisrmf, mkarf, mkwarf, arfcorr, asphist, dmgroup, dmhedit, dmcopy, acis_fef_lookup, sky2tdet, combine_spectra, ardlib, acis_set_ardlib, dmmakereg, add_tool_history, dmhistory, dmimgthresh
```

Reply:
this has in the distant past, but is unused, since the time savings is minimal with parallelization.  I've pulled it.